### PR TITLE
[i18n:eu] Add Euskara language support

### PR DESCRIPTION
**Pull request by darkwing**
Superecedes https://github.com/MetaMask/metamask-extension/pull/12063 to fix lint.

### CODE REVIEW

**Review by kumavis — DISMISSED** [review]

**Comment by mcmire** [comment]
Should probably wait until #13206 is merged before merging this.

**Comment by mcmire** [comment]
@jpuri I think it's okay to have languages not supported by Crowdin. That means that the community will have to maintain them, but I think that's fine. Also, this language shouldn't conflict with any PRs Crowdin gives us. Agree/disagree?

**Review by brad-decker — APPROVED** [review]

**Pull request closed without merging**
